Code/Legacy/CrySystem/System.cpp

```diff
@@ -241,6 +241,9 @@ CSystem::CSystem()
 #endif
 
     m_ConfigPlatform = CONFIG_INVALID_PLATFORM;
+
+    m_movieSystem = nullptr;
+    Maestro::MovieSystemRequestBus::BroadcastResult(m_movieSystem, &Maestro::MovieSystemRequestBus::Events::GetMovieSystem);
```

Comment:
I expect CrySystem/System to be init super early, we're sure that we won't end up in a case where m_movieSystem is always nullptr because Maestro hasn't been init ?

Reply:
It gets set during `OnCrySystemInitialized` which is also where gEnv->pMovieSystem was being populated before, the behavior/timing did not change